docs/dashboards/synchronize-components.md

```diff
@@ -119,3 +119,20 @@ Demo:
 |[DataGrid](https://www.highcharts.com/docs/dashboards/datagrid-component)|yes|yes|yes|no|
 |[KPI](https://www.highcharts.com/docs/dashboards/kpi-component)|no|yes|no|no|
 |[Navigator](https://www.highcharts.com/docs/dashboards/navigator-component)|no|yes|no|yes|
+
+
+## Custom synchronization
+
+For each component, you can add or overwrite a custom [definition of the existing synchronization](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_Sync_Sync.Sync.OptionsEntry) `handler` and/or `emitter` to expand its functionality. You can also define your own synchronization from scratch.
+
+```js
+sync: {
```

Comment:
IMO it doesn't sufficiently explain what the handler/emitter should do.
Please describe more parts:
```ts
        this.emit({
            target: this,
            type: 'sliderValueChanged',
            rowIndex: rowIndex
        });
```

```ts
return this.on('sliderValueChanged', e => {
```

```ts
cursor.emitCursor(table, {
```
and
```ts
cursor.removeListenerr(
```